src/webgpu/api/validation/encoding/cmds/render/indirect_draw.spec.ts

```diff
@@ -0,0 +1,78 @@
+export const description = `
+Validation tests for drawIndirect/drawIndexedIndirect on render pass and render bundle.
```

Comment:
Do we need to test if the draw parameters stored in the indirect buffer are properly validated, just like they are used in ```draw()``` | ```drawIndexed()```?

Reply:
We should validate the indirect buffer parameters like `draw()` and `drawIndexed()` did, but we haven't such validation rules in indirect draw calls, is it missing or not added due to other reasons? @kainino0x

Reply:
The parameters inside the indirect buffer exist only on the GPU, so they have to be validated on-GPU, so they can't produce validation errors without blocking on all previous GPU work. Instead, various behaviors are allowed - no-op the draw call, or robust buffer access: https://github.com/gpuweb/gpuweb/pull/1835

You can also think of it this way: An indirect draw call's parameters going out of bounds (of an index or vertex buffer) is similar to an index buffer going out of bounds (on a vertex buffer).

Reply:
Thank you for your answers! I think we can test the invalid parameters in indrect buffer in operation tests.